lang/hugs98/Portfile

```diff
@@ -1,42 +1,88 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
 
 name                hugs98
 version             plus-Sep2006
-revision            2
-categories          lang
-maintainers         nomaintainer
-platforms           darwin
+revision            3
```

Comment:
Sorry @barracuda156 just noticed that this doesn't warrant a revbump, since it only fixes the build and doesn't modify any installed files. Taking a look at the [stats](https://ports.macports.org/port/hugs98/stats/), the few people that do have hugs98 installed shouldn't have to rebuild it.

Also, it seems like at least three people have (somehow) installed it on macOS 12 and 10.13, but here we're blacklisting for > 10.11

Reply:
> doesn't warrant a revbump, since it only fixes the build and doesn't modify any installed files

We cannot be sure that everything is installed exactly the same, since there are new patches. We already had a case with another port, when avoiding revbump in a silimar situation led into trouble.

> Also, it seems like at least three people have (somehow) installed it on macOS 12 and 10.13, but here we're blacklisting for > 10.11

I have seen that, but it is likely some glitch. Neither buildbots (with current version of the port) nor CIs (with updated one) succeeded to build it on newer OSs.

Reply:
@harens Thank you! Glad this is finally merged.